### PR TITLE
Kacper murzyn/7.32.0 changelog

### PR DESCRIPTION
**Pull request by kacper-murzyn**
### What does this PR do?

Changelog for 7.32.0 agent releases added.

### CODE REVIEW

**Review by bkabrda — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,182 @@
...
+
+
+.. _Release Notes_7.32.0_Upgrade Notes:
+
+Upgrade Notes
+-------------
+
+- APM: Change default profiling intake to use v2 endpoint.
+
+- CSPM the check subcommand is now part of the security-agent compliance.
```
Comment (olivielpeau) [inline CHANGELOG.rst:27]:
```suggestion
- CSPM: the ``check`` subcommand is now under the ``security-agent compliance`` command (it was previously directly under ``security-agent``)
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,182 @@
...
+
+- The "containerd" check is now supported on Windows.
+
+- Add experimental support for writing agent-side CSPM compliance checks in Rego.
+
+- Runtime security can now attach span/trace to event.
+
+- Provides alternative implementation for process collection on Windows using performance counters.
+
+- Add multi-line auto-sensing when tailing logs from file.
```
Comment (olivielpeau) [inline CHANGELOG.rst:59]:
```suggestion
- Add multi-line auto-sensing when tailing logs from file (in beta).
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,182 @@
...
+  It checks the 1000 first lines (or waits 30 seconds, whichever is first)
+  when tailing for a list of known timestamp formats. If the 
+  number of matched lines is greater than the threshold it 
+  switches to the MultiLineHandler with the pattern matching
+  the timestamp format. The pattern chosen is saved in the log
+  config and is reused if the file rotates.  Use the new global config 
+  parameter: ``logs_config.auto_multi_line_detection`` to enable
+  the feature.  New per log integration config parameters:
+  * ``auto_multi_line_detection`` to enable the feature on a case
+    by case basis
```
Comment (olivielpeau) [inline CHANGELOG.rst:69]:
```suggestion
  config and is reused if the file rotates.  Use the new global config 
  parameter ``logs_config.auto_multi_line_detection`` to enable
  the feature for the whole agent, or the per log integration config parameter ``auto_multi_line_detection``
  to enable the feature on a case by case basis.
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,182 @@
...
+
+- Tag Kubernetes containers with ``image_id`` tag.
+
+- Eliminates the need to synchronize state between regular and RT process collection.
+
+- APM: Added a configuration option to set the API key separately for Live
+  Debugger. It can be set via `apm_config.debugger_api_key` or
+  `DD_APM_DEBUGGER_API_KEY`.
+
+- Update EP forwarder config to use intake v2 for ndm metadata
```
Comment (olivielpeau) [inline CHANGELOG.rst:125]:
```suggestion
- Use v2 intake for NDM metadata
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -2,6 +2,182 @@
...
+
+- Remove the `reason` tag from the `kubernetes_state.job.failed` metric to reduce cardinality
```
Comment (olivielpeau) [inline CHANGELOG.rst:127]:
```suggestion
- Remove the ``reason`` tag from the ``kubernetes_state.job.failed`` metric to reduce cardinality
```

**Review by olivielpeau — APPROVED** [review]
Made a few nits, but LGTM overall thanks

File: CHANGELOG.rst (hunk 1 above)
Comment (kacper-murzyn) [reply CHANGELOG.rst:27]: Updated

File: CHANGELOG.rst (hunk 2 above)
Comment (kacper-murzyn) [reply CHANGELOG.rst:59]: Updated

File: CHANGELOG.rst (hunk 4 above)
Comment (kacper-murzyn) [reply CHANGELOG.rst:124]: Updated

File: CHANGELOG.rst (hunk 5 above)
Comment (kacper-murzyn) [reply CHANGELOG.rst:126]: Updated

**Pull request merged**